src/main/webapp/scripts/private-room-script.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Fetches prvivate room and prints the room to console
+ */
+function fetchPrivateRoom() {
+  console.log('Fetching private room.');
+  // TODO: fetch hard-coded private room
+}
+
+// This code loads the IFrame Player API code asynchronously.
+var scriptTag = document.createElement('script');
+
+scriptTag.src = "https://www.youtube.com/iframe_api";
+var firstScriptTag = document.getElementsByTagName('script')[0];
+firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
```

Comment:
move this to top of file. Have all global variables above any global statements (code).

Reply:
Not resolved, please update. If you disagree, please add to conversation here.